astro.config.mjs

```diff
@@ -1,70 +1,62 @@
 import path from 'path';
 import { fileURLToPath } from 'url';
-
 import { defineConfig } from 'astro/config';
-
 import tailwind from '@astrojs/tailwind';
 import sitemap from '@astrojs/sitemap';
 import image from '@astrojs/image';
 import mdx from '@astrojs/mdx';
 import partytown from '@astrojs/partytown';
 import compress from 'astro-compress';
 import { readingTimeRemarkPlugin } from './src/utils/frontmatter.mjs';
-
 import { SITE } from './src/config.mjs';
+import svelte from "@astrojs/svelte";
+// import node from '@astrojs/node';
+// import vercelEdge from '@astrojs/vercel/edge';
+import vercel from "@astrojs/vercel/serverless";
 
 const __dirname = path.dirname(fileURLToPath(import.meta.url));
+const whenExternalScripts = (items = []) => SITE.googleAnalyticsId ? Array.isArray(items) ? items.map(item => item()) : [items()] : [];
 
-const whenExternalScripts = (items = []) =>
-  SITE.googleAnalyticsId ? (Array.isArray(items) ? items.map((item) => item()) : [items()]) : [];
 
+// https://astro.build/config
 export default defineConfig({
   site: SITE.origin,
   base: SITE.basePathname,
   trailingSlash: SITE.trailingSlash ? 'always' : 'never',
-
-  output: 'static',
-
+  output: 'server',
```

Comment:
As this site also has static paths for the blog, etc. it should render as hybrid. If output:'server' all the static paths break

See docs
https://docs.astro.build/en/guides/server-side-rendering/#converting-a-static-site-to-hybrid-rendering

output: 'hybrid'